Makefile

```diff
@@ -531,8 +531,47 @@ CPPLINT_FILES = $(filter-out $(CPPLINT_EXCLUDE), $(wildcard \
 	tools/icu/*.h \
 	))
 
-cpplint:
-	@$(PYTHON) tools/cpplint.py $(CPPLINT_FILES)
+GOOGLE_DOWNLOAD ?= tools/format/download_from_google_storage
+GOOGLE_DOWNLOAD_FLAGS ?= --no_resume --no_auth
+
+tools/format/clang-format-mac: tools/format/clang-format-mac.sha1
+	@echo "Downloading clang-format, may take some time"
+	@$(GOOGLE_DOWNLOAD) $(GOOGLE_DOWNLOAD_FLAGS) --platform=darwin \
+		--bucket chromium-clang-format -s $<
+
+tools/format/clang-format-win.exe: tools/format/clang-format-win.exe.sha1
+	@echo "Downloading clang-format, may take some time"
+	@$(GOOGLE_DOWNLOAD) $(GOOGLE_DOWNLOAD_FLAGS) --platform=win32 \
+		--bucket chromium-clang-format -s $<
+
+tools/format/clang-format-linux64: tools/format/clang-format-linux64.sha1
+	@echo "Downloading clang-format, may take some time"
+	@$(GOOGLE_DOWNLOAD) $(GOOGLE_DOWNLOAD_FLAGS) --platform=linux* \
+		--bucket chromium-clang-format -s $<
+
+ifeq ($(OSTYPE),darwin)
+CLANG_FORMAT ?= tools/format/clang-format-mac
+else ifeq ($(OSTYPE),linux)
+CLANG_FORMAT ?= tools/format/clang-format-linux64
+else
+CLANG_FORMAT ?= tools/format/clang-format-win.exe
+endif
+
+CLANG_LINT ?= ./tools/format/git-clang-format --binary $(CLANG_FORMAT)
+CLANG_LINT_COMMIT ?= 81e32cc
```

Comment:
Shouldn't that be something like `CLANG_LINT_COMMIT ?= $(shell $(GIT) rev-parse origin/master)`?